### ISSUE TITLE
Figure out a shortcut for dark mode

### ISSUE THREAD

**Issue by dwelle** [excalidraw/excalidraw#3052]
Any standard out there? Couldn't find.

**Comment by h7y** [comment]
I use this extension called [Dark Reader](https://addons.mozilla.org/en-US/firefox/addon/darkreader/) for adding dark mode to any site.  For dark mode toggle, `Option or Alt + Shift + d` is used.

**Comment by lipis** [comment]
How often do people change the theme so we need a shortcut for it? I think we just need to comply on load #2025

**Comment by dwelle** [comment]
Yea, if there's no common standard, or we don't figure out something obvious, we can prob skip it.

> I use this extension called [Dark Reader](https://addons.mozilla.org/en-US/firefox/addon/darkreader/) for adding dark mode to any site. For dark mode toggle, `Option or Alt + Shift + d` is used.

Interesting, `Alt + Shift + d` was what I came up with, too. The only downside is it's not mac-friendly in terms of keyboard layouts (we'd need to use `KeyboardEvent.code` instead of `.key` coz Mac prints differents keys when using the `Option` modifier).

**Comment by ad1992** [comment]
We can still add a shortcut key as this would enhance the usability of keyboard users. 
`alt+shift+d` sounds good. We are anyways using `option+z / option+r` so should be fine to use `KeyboardEvent.code`

**Comment by h7y** [comment]
I can take this up if we are adding the shortcut!

**Comment by dwelle** [comment]
I think `Alt+Shift+D` is a good shortcut. We're already using it in Excalidraw+. @h7y taking it on?

**Comment by h7y** [comment]
Sure!
